xformers/components/feedforward/mlp.py

```diff
@@ -34,13 +38,27 @@ def __init__(
     ):
         super().__init__()
 
-        self.mlp = nn.Sequential(
-            nn.Linear(dim_model, hidden_layer_multiplier * dim_model, bias=bias),
-            build_activation(activation),
-            nn.Dropout(dropout),
-            nn.Linear(hidden_layer_multiplier * dim_model, dim_model, bias=bias),
-            nn.Dropout(dropout),
-        )
+        dim_mlp = hidden_layer_multiplier * dim_model
+
+        if activation in {Activation.ReLU, Activation.GeLU} and _is_functorch_available:
+            self.mlp = nn.Sequential(
+                nn.Linear(dim_model, hidden_layer_multiplier * dim_model, bias=False),
+                FusedBiasActivationDropout(
```

Comment:
nit but may be a bit cleaner if we just define the FusedBiasActiationDropout in the if/else and then build the mlp after?